lib/src/parsers/utilities/expression-parser.ts

```diff
@@ -22,6 +22,19 @@ export function parseExpression(expression: Expression): DataExpression {
     return booleanExpression(expression.getLiteralValue());
   } else if (TypeGuards.isStringLiteral(expression)) {
     return stringExpression(expression.getLiteralValue());
+  } else if (TypeGuards.isPrefixUnaryExpression(expression)) {
+    if (expression.getOperatorToken() === SyntaxKind.MinusToken) {
```

Comment:
I'd suggest a `switch` statement instead, and handling the plus token as well:

```
const operand = expression.getOperand();
if (!TypeGuards.isNumericLiteral(operand)) {
  throw new Error("unary operators may only be used with numeric literals");
}
switch (expression.getOperatorToken()) {
  case SyntaxKind.MinusToken:
    return numberExpression(-operand.getLiteralValue());
  case SyntaxKind.PlusToken:
    return numberExpression(+operand.getLiteralValue());
  default:
    throw new Error("unknown prefix operator token");
}
```

Reply:
is `age: +30` syntax we want to support?

Reply:
I think so, at least I see no reason to support -30 but not support +30. It can be useful for example when you have two tests with opposite signs, and you want to make it clear that the difference is the sign. See what I mean?